ddtrace/contrib/django/conf.py

```diff
@@ -193,8 +194,9 @@ def configure_from_settings(pin, config, settings):
     if "ENABLED" in settings:
         pin.tracer.enabled = settings["ENABLED"]
 
-    if "AGENT_HOSTNAME" in settings:
-        pin.tracer.writer.api.hostname = settings["AGENT_HOSTNAME"]
-
-    if "AGENT_PORT" in settings:
-        pin.tracer.writer.api.port = settings["AGENT_PORT"]
+    if "AGENT_HOSTNAME" in settings or "AGENT_PORT" in settings:
+        # WTF changing the URL like that?
```

Comment:
This is for migration from tracer configuration in settings module, which has been deprecated.